project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   val nel       = "com.evolutiongaming" %% "nel"       % "1.3.5"
   val playJson  = "com.typesafe.play"   %% "play-json" % "2.10.7"
   val scalaTest = "org.scalatest"       %% "scalatest" % "3.2.19"
-  val jsoniter  = "com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-core"  % "2.23.0"
+  val jsoniter  = "com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-core"  % "2.36.4"
```

Comment:
let's use latest version:
```suggestion
  val jsoniter  = "com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-core"  % "2.36.7"
```